packages/react/src/ReactStartTransition.js

```diff
@@ -9,9 +9,20 @@
 
 import ReactCurrentBatchConfig from './ReactCurrentBatchConfig';
 
+// This should stay in sync with the reconciler (ReactEventPriorities).
+// Intentionally not using a shared module, because this crosses a package
+// boundary: importing from a shared module would give a false sense of
+// DRYness, because it's theoretically possible for for the renderer and
+// the isomorphic package to be out of sync. We don't fully support that, but we
+// should try (within reason) to be resilient.
```

Comment:
On the other hand, copying over gives a false sense of safety since if the renderer and isomorphic are out of sync, this could also be wrong.